jsx/Form.js

```diff
@@ -2221,9 +2221,9 @@ class StaticElement extends Component {
       <div className="row form-group">
         {label}
         <div className={this.props.class}>
-          <p className={this.props.textClass}>
+          <div className={this.props.textClass}>
```

Comment:
Why is this changed? I think `<p>` is probably more accurate (it seems to be a paragraph in this context..) and changing Form.js affects every place that uses it.

Reply:
The text can contain another p which leads to an invalid nesting of p.